src/app/core/token-interceptor.service.ts

```diff
@@ -0,0 +1,24 @@
+import {Inject, Injectable} from '@angular/core';
+import {HttpEvent, HttpHandler, HttpRequest} from '@angular/common/http';
+import {Observable} from 'rxjs';
+import {AuthConfig, AuthTokenConfig} from '@core';
+
+@Injectable()
+export class TokenInterceptorService {
+
+  constructor(@Inject(AuthConfig) private config: AuthTokenConfig) {
```

Comment:
Again: do not inject the AuthTokenConfig inside multiple services, use `AuthService` instead.